packages/flp-config-sub-generator/test/app.test.ts

```diff
@@ -434,8 +434,8 @@ describe('flp-config generator', () => {
         assertInboundsHasConfig(get(changedManifest, crossNavigationPropertyPath), answers, true);
 
         const i18nContent = fs.readFileSync(`${OUTPUT_DIR_PREFIX}/app1/webapp/i18n/i18n.properties`).toString();
-        expect(i18nContent).toContain('flpTitle=title1');
-        expect(i18nContent).toContain('flpSubtitle=subtitle1');
+        expect(i18nContent).toContain(`flpTitle=${answers.semanticObject}-${answers.action}-${answers.title}`);
```

Comment:
This doesn't seem right. 

Should it not be `${answers.semanticObject}-${answers.action}-flpTitle`=${answers.title}` ?

Reply:
Thank you for the review. It concatenates it for you, whereas you only type in the title.

Reply:
Sorry I don't understand what you mean. 
This is adding the semanticObect and action to the value. Whereas it should be added to the key

Reply:
Sorry my bad, you are right. Thanks